src/planners/trajectory/Spline5.cpp

```diff
@@ -255,6 +255,38 @@ bool planning::trajectory::Spline5::checkConstraints(int idx, float t_f)
     return true;
 }
 
+std::vector<float> planning::trajectory::Spline5::getMaxVelocityTimes(int idx)
+{
+    return solveQubicEquation(20*a(idx), 12*b(idx), 6*c(idx), 2*d(idx));
+}
+
+std::vector<float> planning::trajectory::Spline5::getMaxAccelerationTimes(int idx)
+{
+    std::vector<float> t_max;
+
+    if (a(idx) != 0)
+    {
+        float D = 576*b(idx)*b(idx) - 1440*a(idx)*c(idx);
```

Comment:
All magic numbers should be declared as constants.

Reply:
These magic numbers are very specific for each type of spline, e.g., these ones are for quintic spline (Spline5), and they will never be changed, and they cannot be changed. On the other hand, after we implement Spline3 or Spline4, they will have their unique magic numbers. However, there are too many magic numbers in Spline5, thus I don't know how to name all of them. :D